app/controllers/admin/surveys_controller.rb

```diff
@@ -1,10 +1,13 @@
 module Admin
   class SurveysController < Admin::BaseController
     def index
-      @closed_surveys = Survey.where(closed: true)
-      @opened_surveys = Survey.where(closed: false)
+      @closed_surveys = policy_scope(Survey).where(closed: true)
```

Comment:
it looks like `policy_scope` is necessary, because the admin expects to have access to all data

Reply:
solved